server/sender.py

```
@@ -44,6 +55,81 @@ def run_process(cmd: list[str]) -> subprocess.Popen:
 
+#: טבלאות ה-UDP של הקרנל — מקור האמת למי מחזיק פורט בלינוקס.
+UDP_TABLES = (Path("/proc/net/udp"), Path("/proc/net/udp6"))
+
+#: פורט תפוס שלא הצלחנו לשייך לתהליך (סוקט של משתמש אחר). "לא ידוע מי"
+#: אינו "אין אף אחד" — ולכן זה מחזיק, לא פנוי.
+UNKNOWN_HOLDER = -1
+
+
+def _inodes_on_port(port: int) -> set[str] | None:
+    """ה-inodes של סוקטי UDP שקשורים ל-`port`; ‏None כשאין טבלה לקרוא."""
+    inodes: set[str] = set()
```

Ruff (0.16.2)

[error] Contains control characters that can permit obfuscated code

(PLE2502)